src/analysis/retail/priest/shadow/modules/guide/ExtraComponents.tsx

```diff
@@ -0,0 +1,20 @@
+import styled from '@emotion/styled';
+import { QualitativePerformance } from 'parser/ui/QualitativePerformance';
+import { ComponentProps, ReactNode } from 'react';
+import { PerformanceMark, qualitativePerformanceToColor } from 'interface/guide';
+
+export const ColoredStrong = styled.strong`
+  color: ${(props) => props.color};
+`;
+
+interface PerformanceStrongProps extends ComponentProps<typeof ColoredStrong> {
+  children: ReactNode;
+  performance: QualitativePerformance;
+}
+export const PerformanceStrong = ({ performance, ...others }: PerformanceStrongProps) => (
```

Comment:
I want to move this to a shared place instead of copying it between specs. It doesn't need to be fixed in this PR.

Reply:
That makes sense.  Whenever it is moved to a shared place I will update shadow to use that.